website/docs/tutorials/06-deploying/02-migrations/invoke.md

```diff
@@ -18,8 +18,7 @@ It's an extension of [CheatcodeNetworkConfig](../03-network-config.md), so it's
 ```python
 Wei = int
 class SignedCheatcodeConfig(CheatcodeNetworkConfig):
-    max_fee: Wei | None
-    auto_estimate_fee: int
+    max_fee: Fee
```

Comment:
`Wei` type is not used, and it should be replaced with `Fee`.

Reply:
[fix docs](https://github.com/software-mansion/protostar/pull/979/commits/794612a37db566139eaf1c36bad0577a0d6e83fc)